Project.toml

```diff
@@ -0,0 +1,19 @@
+name = "StatsFuns"
+uuid = "4c63d2b9-4356-54db-8cca-17b64c39e42c"
+version = "0.9"
```

Comment:
```suggestion
version = "0.9.0"
```
?

Reply:
Isn't that effectively the same thing?

Reply:
Yeah I suppose so, I just don't usually see packages declare their own versions that way.